council/spending_proposal_categories.csv

```diff
@@ -1390,3 +1390,24 @@ Giza,1167,,,,,,,,,,
 Giza,1168,,,,,,,,,,
 Giza,1169,,,,,,,,,,
 Giza,1170,Payments for Bounty #24 (20th Week),Approved,12870012,12870012,0,00002331,300,Bounties,24,,N
```

Comment:
> It looks like row 1392 should actually have 12 columns, instead of 13. in line 1391.
https://github.com/Joystream/community-repo/blob/cf8fc268e2547bdc0f9d32b860f673a44677ab1f/council/spending_proposal_categories.csv

Reply:
resolved in 6db96e7